api_tests/lib/wallets/lightning.ts

```diff
@@ -14,7 +13,7 @@ import { AddressType } from "@radar/lnrpc";
 export class LightningWallet implements Wallet {
     public static async newInstance(
         bitcoinWallet: BitcoinWallet,
-        logger: Logger,
+        // logger: Logger,
```

Comment:
```suggestion
```

Reply:
Ups :D

Yeah those should be activated again :)